src/lektorium/repo/local/storage.py

```diff
@@ -169,6 +174,56 @@ def prepare(cls, site_config):
         return site_config
 
 
+class GitLab:
+    def __init__(self, options):
+        self.options = options
+
+    @cached_property
+    def projects(self):
+        response = requests.get(
+            '{scheme}://{host}/api/v4/projects'.format(**self.options),
```

Comment:
api version could be a class const

Also `{scheme}://{host}/api/v4/projects` repeats and could be extracted into a property